data/json/effects.json

```diff
@@ -460,6 +481,31 @@
     },
     {
         "type": "effect_type",
+         "id": "ooze_in_eyes",
+        "name": ["Gunk in eyes"],
+        "desc": ["There is some sort of substance in your eyes. You can barely see anything now!"],
+        "apply_message": "Your eyes sting and you can keep them barely open!",
+        "rating": "bad",
+        "base_mods": {
+            "per_mod": [-6]
+        }
+    },
+    {
+        "type": "effect_type",
+        "id": "inked",
+        "name": ["Soaked in ink"],
+        "desc": ["You are covered in black ink!"],
+        "apply_message": "You're covered in black ink!",
+        "int_dur_factor": 300,
```

Comment:
This needs a max intensity.

Smell of ink isn't neutral (unlike say, flowery smell) and so can be used to track too.
This means that once you smell like ink, you are still trackable and extra ink won't help here.